### PR TITLE
docs(master): fix init container not support lifecycle and probe

### PR DESCRIPTION
**Pull request by cuisongliu**
Hello Kubernetes Community!

This PR fixes init container not support lifecycle and probe in language/zh documentation. 

Have a nice day

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ If you specify multiple init containers for a Pod, Kubelet runs each init contai
 
 Init 容器支持应用容器的全部字段和特性，包括资源限制、数据卷和安全设置。 然而，Init 容器对资源请求和限制的处理稍有不同，在下面 [资源](#资源) 处有说明。
 
-同时 Init 容器不支持 Readiness Probe，因为它们必须在 Pod 就绪之前运行完成。
+同时 Init 容器不支持 Lifecycle、LivenessProbe、ReadinessProbe和StartupProbe，因为它们必须在 Pod 就绪之前运行完成。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/init-containers.md:77]: pls also sync English content on line 69.

Comment (cuisongliu) [reply content/zh/docs/concepts/workloads/pods/init-containers.md:77]: ok. thanks, I am done it .

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 2]
```diff
@@ -66,15 +66,15 @@ Init 容器的状态在 `status.initContainerStatuses` 字段中以容器状态
...
+Also, init containers do not support `lifecycle`, `livenessProbe`, `readinessProbe`, or `startupProbe` because they must run to completion before the Pod can be ready.
 If you specify multiple init containers for a Pod, Kubelet runs each init container sequentially. Each init container must succeed before the next can run. When all of the init containers have run to completion, Kubelet initializes the application containers for the Pod and runs them as usual.
 -->
 
 ### 与普通容器的不同之处
 
 Init 容器支持应用容器的全部字段和特性，包括资源限制、数据卷和安全设置。 然而，Init 容器对资源请求和限制的处理稍有不同，在下面 [资源](#资源) 处有说明。
 
-同时 Init 容器不支持 Readiness Probe，因为它们必须在 Pod 就绪之前运行完成。
+同时 Init 容器不支持 Lifecycle、LivenessProbe、ReadinessProbe和StartupProbe，因为它们必须在 Pod 就绪之前运行完成。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/init-containers.md:77]:
```suggestion
同时 Init 容器不支持 `lifecycle`、`livenessProbe`、`readinessProbe` 和 `startupProbe`，因为它们必须在 Pod 就绪之前运行完成。
```

**Pull request merged**